Mathlib/Order/Interval/Set/Basic.lean

```diff
@@ -38,37 +38,6 @@ section Preorder
 
 variable [Preorder α] {a a₁ a₂ b b₁ b₂ c x : α}
 
-/-- `Ioo a b` is the left-open right-open interval $(a, b)$. -/
```

Comment:
The module docstring still talks about "contains these definitions" - this should be updated, right?

Reply:
Done.